### DISCUSSION TITLE
Budibase 3 and table permissions

### DISCUSSION THREAD

**Discussion by aowdnmp** [Budibase/budibase#14973] [General]
Before budibase 3 upgrade I was having an option to decide user groups that would have been able to access each table data in Read (ie. app users) or Read/Write (ie. app admins). Now, having upgraded. I can only set the group who has readwrite permissions, so I'm not able to distinguish this anymore? Or am I missing something? Thanks

**Comment by mike12345567** [comment]
Hi @aowdnmp - theres a discussion about how this works here: https://github.com/Budibase/budibase/discussions/10563#discussioncomment-11164646

Essentially this allows for more granular control, you can hide or set columns to read-only, creating a view which only allows reading the columns in it.

**Reply by saz33m1** [reply]
@mike12345567 what If I want a table to writable by public, but readable by only admins?

**Reply by aowdnmp** [reply]
@mike12345567 This way, using views, do I need to publish different screens for different group of users? One screen with the table for admins and one screen with readonly view for users? Before I was having just one table screen for both. Do I understand correctly? Thanks

PS. btw I see that in views, for display columns and required columns is not possible to set them as readonly...

**Reply by SimonDoll** [reply]
> @mike12345567 what If I want a table to writable by public, but readable by only admins?

This is exactly my use case and I'm wondering how to achieve this with budibase 3.

The visitor-check-in-form template provided with budibase is a good example to showcase the issue @mike12345567
